src/app.ts

```diff
@@ -35,19 +35,25 @@ declare global {
   namespace Express {
     interface Request {
       generatedSessionId?: string;
+      cri?: {
+        id?: string;
+        redirectUrl?: string;
+      };
+      redirectURL?: URL;
     }
   }
 }
 
 // Extend session object with properties we expect
 declare module "express-session" {
   interface SessionData {
-    ipvSessionId: string;
-    clientOauthSessionId?: string;
+    ipvSessionId: string | null;
+    clientOauthSessionId?: string | null;
```

Comment:
In the ipv middleware (line 136), they are being set to null for some reason. I wasn't sure if setting it to null was intentional or not so I left as null to avoid potentially breaking something further down the line